core/indigo-core/molecule/src/molecule_cdxml_saver.cpp

```diff
@@ -1520,6 +1520,11 @@ void MoleculeCdxmlSaver::endDocument()
 {
     if (_is_binary)
     {
+        std::ifstream t("C:\\cdx\\benz.xml");
```

Comment:
That is definitely something that should not exist in this pull request.

Reply:
removed